src/app/lib/utilities/getEnvConfig/index.ts

```diff
@@ -0,0 +1,46 @@
+import { Environments } from '#app/models/types/global';
+import onClient from '../onClient';
+
+export type EnvConfig = ReturnType<typeof getProcessEnvAppVariables>;
+
+// Any environment variables added here will be available to the client and server
+export const getProcessEnvAppVariables = () => ({
```

Comment:
Per my comment below, is there some way we could ensure that these environment variables match what's in our local/test/live environment files? Just to avoid issues if a dev adds a new environment variable but forgets to add it here. Could typescript help us somehow?

Reply:
We could convert the `.env` files to JSON objects and type check them? The only problem is this would only apply to the Express app, as the `.env` files for the Next.js app are set ultimately set and read from the Lambda function itself.